internal/api/api.go

```diff
@@ -17,17 +17,16 @@ package api
 import (
 	"fmt"
 	"net/http"
-	"time"
 
-	ginzap "github.com/gin-contrib/zap"
+	"log/slog"
```

Comment:
You'll need to update the `.github/workflows/go-test.yml` and `.github/workflows/publish.yml` to use 1.21/1.22 (build using 1.21 for now) if you want to update this. Personally, I would split the logging changes from the CLI changes and do this in another pull request.